src/components/CaptureFilter.js

```diff
@@ -119,6 +132,59 @@ function Filter(props) {
     return convertDateToDefaultSqlDate(date);
   };
 
+  const handleVerificationStatusChange = (event) => {
+    const value = event.target.value;
+    if (
+      value[value.length - 1] === 'all' ||
+      (value.length === 3 && value[value.length - 1] !== 'all')
+    ) {
+      setVerificationStatus(
+        verificationStatus.length === verificationStatesArr.length
+          ? []
+          : verificationStatesArr,
+      );
+      setActive(undefined);
+      setApproved(undefined);
+      return;
+    }
+    handleChangeActiveApproved(value);
+    setVerificationStatus(value);
+  };
+
+  // set active and approved field based on the checkbox values
+  function handleChangeActiveApproved(value) {
```

Comment:
Unfortunately, this logic doesn't cover all possible combinations, and it's hard to follow what it's trying to do.
If _Approved_ and _Rejected_ are selected, everything (including _Awaiting Verification_) is included.

I'd suggest replacing `approved` and `active` in the filter model with these Verification States and only converting when the API query is constructed.
You will need to apply an `and` clause to the query object: https://loopback.io/doc/en/lb4/Where-filter.html#and--or